backend/app/serializers/answers_survey_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class AnswersSurveySerializer < ActiveModel::Serializer
+  attributes :id, :survey_id, :user_id
```

Comment:
Not sure if we talk about this, but now, looks like it will be useful to have a survey instead of survey_id, it will save frontend of doing one more request, what do you think?

Reply:
Yes makes much more sense! but do we need the serializer as we won't use the index?

Reply:
yes, we do need, the serializer is used by the `create` method in the controller

Reply:
True!